cmd/proxy/config/swagger/openapi.json

```diff
@@ -0,0 +1,3016 @@
+      "get": {
+        "tags": [
+          "status"
+        ],
+        "summary": "returns the same metrics as /status/metrics but in Prometheus format",
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/GenericResponse"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/actions/reload-observers": {
+      "post": {
+        "tags": [
+          "actions"
+        ],
+        "summary": "will reload the observers configuration based on the updated config.toml file. REQUIRES AUTHENTICATION",
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/GenericResponse"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/actions/reload-full-history-observers": {
+      "post": {
+        "tags": [
+          "actions"
+        ],
+        "summary": "will reload the full history nodes configuration based on the updated config.toml file. REQUIRES AUTHENTICATION",
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/GenericResponse"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/node/heartbeatstatus": {
+      "get": {
+        "tags": [
+          "node"
+        ],
+        "summary": "returns the heartbeat data from an observer from any shard. Has a cache to avoid many requests",
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/GenericResponse"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/node/old-storage-token/{token}/nonce/{nonce}": {
+      "get": {
+        "tags": [
+          "node"
+        ],
+        "summary": "will return a boolean value indicating if the token's metadata is still stored in accounts' trie, as opposed to the system account",
+        "parameters": [
+          {
+            "name": "token",
+            "in": "path",
+            "description": "the token to look after",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            }
+          },
+          {
+            "name": "nonce",
+            "in": "path",
+            "required": true,
+            "description": "the nonce to look after",
+            "schema": {
+              "type": "string"
```

Comment:
not sure about types here, interger for nonce and string for token?

Reply:
yup, changed